src/types/redux/captures.ts

```diff
@@ -73,7 +73,8 @@ export type CaptureActions =
 export type CaptureState = {
   document_front?: DocumentCapture & MetadataState
   document_back?: DocumentCapture & MetadataState
-  document?: DocumentCapture & MetadataState
+  // when is this used
+  // document?: DocumentCapture & MetadataState
```

Comment:
Don't think this is ever used anywhere since we still send `document_front` for a 1-sided document.